### PR TITLE
AsyncValidation - Sync validation options

### PR DESCRIPTION
**Pull request by NickMitrokhin**
1. Fixes rendering pending indicator in lookup.
2. Synchronizes isValid, validationStatus, validationErrors and validationError options.

### CODE REVIEW

**Review by mrjohn0011 — DISMISSED** [review]

File: js/ui/editor/editor.js [hunk 1]
```diff
@@ -122,9 +127,9 @@ const Editor = Widget.inherit({
             /**
             * @name EditorOptions.validationStatus
             * @type Enums.ValidationStatus
-            * @default "valid"
+            * @default VALIDATION_STATUS_VALID
```
Comment (San4es) [inline js/ui/editor/editor.js]:
```suggestion
            * @default "valid"
```

File: js/ui/editor/editor.js [hunk 2]
```diff
@@ -42,8 +45,9 @@ const getValidationErrorMessage = function(validationErrors) {
 * @hidden
 */
 const Editor = Widget.inherit({
-    ctor: function() {
+    ctor: function(_, options) {
         this.showValidationMessageTimeout = null;
+        this._initialValidationConfig = options;
```
Comment (San4es) [inline js/ui/editor/editor.js]: Move to `_initOptions()`?

File: js/ui/editor/editor.js [hunk 3]
```diff
@@ -57,6 +61,7 @@ const Editor = Widget.inherit({
         if($element) {
             dataUtils.data($element[0], VALIDATION_TARGET, this);
         }
+        this._initValidationOptions();
```
Comment (San4es) [inline js/ui/editor/editor.js]: Move to `_initOptions()`?

File: js/ui/editor/editor.js (hunk 2 above)
Comment (NickMitrokhin) [reply js/ui/editor/editor.js]: OK

File: js/ui/editor/editor.js (hunk 3 above)
Comment (NickMitrokhin) [reply js/ui/editor/editor.js]: OK

**Review by San4es — APPROVED** [review]

**Pull request merged**